cmake/pcl_find_vtk.cmake

```diff
@@ -87,7 +87,9 @@ else()
   list(APPEND PCL_VTK_COMPONENTS RenderingContext${VTK_RENDERING_BACKEND})
 endif()
 
+message("WITH_QT is:${WITH_QT}")
```

Comment:
Maybe add a space after the colon 🙂

Reply:
It was more for 'debugging' - but I guess the information is nice to have, but could probably be consolidated to a single line summerizing most of the VTK settings 😄

Reply:
There was an issue about producing a full-blown CMake configuration report for PCL, maybe it's the kind of information that could go there when it happens.

Reply:
https://cmake.org/cmake/help/v3.17/module/FeatureSummary.html

Reply:
That's something nice and available in 3.5 too (Ubuntu 16.04's CMake version)

@Morwenn is there a issue for this CMake reporting?

Reply:
@kunaltyagi I thought so but apparently it was merely a comment in issue #3603, my memory failed me. Such a summary is definitely something valuable when dealing with build issues (and especially good when users report issues, you don't have to ask again and again for the information). I didn't know that CMake had a built-in feature to report the options, that's nice.